ethereum/executionclient/src/main/java/tech/pegasys/teku/ethereum/executionclient/web3j/Web3jClientBuilder.java

```diff
@@ -31,8 +33,8 @@ public class Web3jClientBuilder {
   private URI endpoint;
   private Optional<JwtConfig> jwtConfigOpt = Optional.empty();
   private Duration timeout;
-
   private ExecutionClientEventsChannel executionClientEventsPublisher;
+  private Collection<String> nonCriticalMethods = new HashSet<>();
```

Comment:
Think we can add already engine_exchangeTransitionConfigurationV1 and engine_exchangeCapabilities. Wdyt?

Reply:
I will add them where we use the constructor instead of as default values. But thanks for the reminder! :)